.mk/k8s.mk

```diff
@@ -75,25 +100,30 @@ k8s-deployonly: $(addsuffix -deployonly,$(addprefix k8s-,$(DEPLOYS)))
 .PHONY: k8s-jaeger-deploy
 k8s-jaeger-deploy:  k8s-start k8s-config k8s-jaeger-delete
 	@until ! $$(kubectl get pods | grep -q ^jaeger ); do echo "Wait for jaeger to terminate"; sleep 1; done
-	@sed "s;\(image:[ \t]*networkservicemesh/[^:]*\).*;\1$${COMMIT/$${COMMIT}/:$${COMMIT}};" ${K8S_CONF_DIR}/jaeger.yaml | kubectl apply -f -
+	@sed "s;\(image:[ \t]*\)\(networkservicemesh\)\(/[^:]*\).*;\1${CONTAINER_REPO}\3$${COMMIT/$${COMMIT}/:$${COMMIT}};" ${K8S_CONF_DIR}/jaeger.yaml | kubectl apply -f -
 
 .PHONY: k8s-admission-webhook-deploy
 k8s-admission-webhook-deploy:  k8s-start k8s-config k8s-admission-webhook-delete k8s-admission-webhook-load-images k8s-admission-webhook-create-cert
 	@until ! $$(kubectl get pods | grep -q ^admission-webhook ); do echo "Wait for admission-webhook to terminate"; sleep 1; done
-	@sed "s;\(image:[ \t]*networkservicemesh/[^:]*\).*;\1$${COMMIT/$${COMMIT}/:$${COMMIT}};" ${K8S_CONF_DIR}/admission-webhook.yaml | kubectl apply -f -
+	@sed "s;\(image:[ \t]*\)\(networkservicemesh\)\(/[^:]*\).*;\1${CONTAINER_REPO}\3$${COMMIT/$${COMMIT}/:$${COMMIT}};" ${K8S_CONF_DIR}/admission-webhook.yaml | kubectl apply -f -
 	@echo "Installing webhook..."
 	@cat ./k8s/conf/admission-webhook-cfg.yaml | ./scripts/webhook-patch-ca-bundle.sh | kubectl apply -f -
 
+.PHONY: k8s-vpn-gateway-nse-deploy
+k8s-vpn-gateway-nse-deploy: k8s-start k8s-config k8s-%-delete k8s-%-load-images
+	@until ! $$(kubectl get pods | grep -q ^vpn-gateway-nse ); do echo "Wait for vpn-gateway-nse to terminate"; sleep 1; done
+	@sed "s;\(image:[ \t]*\)\(networkservicemesh\)\(/icmp-responder-nse[^:]*\).*;\1${CONTAINER_REPO}\3$${COMMIT/$${COMMIT}/:$${COMMIT}};" ${K8S_CONF_DIR}/vpn-gateway-nse.yaml | kubectl apply -f -
+
```

Comment:
Is this a GKE specific piece? If not, please split in separate PR.

Reply:
@edwarnicke did you remember why you was need this part?

Reply:
@nickolaev Here's the deal... you need to get the containers you've built to the K8s cluster somehow... the most expeditious is to build them in Google Cloud Builder and get them from a repo there.  This just provides the extra knobs to enable that.  I suspect we'd do this for every public cloud... but this mechanism should work for all of them.

Reply:
This is generic, yes, I've also updated integration tests to use same environment variable to fix pods deployed.